types/ej.web.all/ej.web.all-tests.ts

```diff
@@ -17,8 +17,8 @@ module AccordionComponent {
         });
     });
 }
-
-
+/// <reference path="../tsfiles/jquery.d.ts" />
```

Comment:
these references are incorrect:
1. they are not at the start of the file, so they don't do anything
2. DefinitelyTyped doesn't have a directory named `tsfiles`.

Reply:
In fact, since this is the only change to the test file besides adding a lot of incorrect spacing, I'd just revert this entire file.